### PR TITLE
CrossPlatform Support (WIP)

### PR DESCRIPTION
**Pull request by uatec**

### CODE REVIEW

File: NuKeeper/ProcessRunner/UnixProcess.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Diagnostics;
+using System.Threading.Tasks;
+
+namespace NuKeeper.ProcessRunner
+{
+    public class UnixProcess : IExternalProcess
```
Comment (AnthonySteele) [inline NuKeeper/ProcessRunner/UnixProcess.cs]: At this stage IMHO a better set of names would be `WindowsExternalProcess` and `UnitExternalProcess`

Comment (uatec) [reply NuKeeper/ProcessRunner/UnixProcess.cs]: Agreed.

Comment (AnthonySteele) [reply NuKeeper/ProcessRunner/UnixProcess.cs]: Er, I mean `UnixExternalProcess`.

File: NuKeeper/NuGet/Process/DotNetUpdatePackageCommand.cs [hunk 2]
```diff
@@ -18,24 +18,21 @@ public class DotNetUpdatePackageCommand : IUpdatePackageCommand
...
-            var updateCommand = $"cd {dirName}"
-                + $" & dotnet restore {sources}"
-                + $" & dotnet remove package {currentPackage.Id}"
-                + $" & dotnet add package {currentPackage.Id} -v {newVersion} -s {packageSource}";
-            _logger.Verbose(updateCommand);
 
-            await _externalProcess.Run(updateCommand, true);
+            await _externalProcess.Run(dirName, "dotnet", $"restore {sources}", true);
+            await _externalProcess.Run(dirName, "dotnet", $"remove package {currentPackage.Id}", true);
+            await _externalProcess.Run(dirName, "dotnet", $"add package {currentPackage.Id} -v {newVersion} -s {packageSource}", true);
```
Comment (skolima) [inline NuKeeper/NuGet/Process/DotNetUpdatePackageCommand.cs]: The logger output has been lost with this change, I think.

**Comment by AnthonySteele** [comment]
replaced by #223
Most of the same process runner code. No merge conficts

**Pull request closed without merging**